sdk/v1/data.go

```diff
@@ -68,8 +70,8 @@ func NewIntegration(name string, version string, arguments interface{}) (*Integr
 		ProtocolVersion:    "1",
 		IntegrationVersion: version,
 		Inventory:          make(Inventory),
-		Metrics:            make([]*metric.MetricSet, 0),
-		Events:             make([]*Event, 0),
+		Metrics:            make([]metric.MetricSet, 0),
```

Comment:
I don't know if we should do the breaking change in protocol 1, if we're going to deprecate it.

Reply:
But well, the breaking change was already done for events. So maybe we shouldn't care.

Reply:
Let's discuss it later in meetings.